### DISCUSSION TITLE
General reviews

### DISCUSSION THREAD

**Discussion by Morzan0521** [DevBase-Dev/twigbot#34] [General]
Hey, found your repo on the explore page. Pretty cool looking bot. However, I noticed your coming-soon/eval.py doesn't appear to have any checks in place. I would highly recommend looking into this as Eval can access _anything_ on your host including your bots token, login info, etc.

**Comment by Morzan0521** [comment]
> Hey, found your repo on the explore page. Pretty cool looking bot. However, I noticed your coming-soon/eval.py doesn't appear to have any checks in place. I would highly recommend looking into this as Eval can access _anything_ on your host including your bots token, login info, etc.

Never mind just noticed it's discontinued. Too bad....

**Comment by VincentRPS** [comment]
@moros0741 Hey! i am becoming more interested to continue this project again if you would like to help you could

**Reply by Morzan0521** [reply]
Hey Vincent! I think its awesome you are interested in getting back into this project! Lemme know if you want any help with anything, and i would be happy to see what i can do. I am currently occupied with my own personal project. If you want to talk more indepth you can speak with me there [Guardian System Discord](https://dsc.gg/dgsofficial)
